lib/ansiblelint/rules/__init__.py

```diff
@@ -245,11 +245,37 @@ def __repr__(self) -> str:
                           for rule in sorted(self.rules, key=lambda x: x.id)])
 
     def listtags(self) -> str:
+        tag_desc = {
+            "behaviour": "Indicates a bad practice or behavior",
+            "bug": "Likely wrong usage pattern",
+            "command-shell": "Specific to use of command and shell modules",
+            "core": "Related to internal implementation of the linter",
+            "deprecations": "Indicate use of features that are removed from Ansible",
+            "experimental": "Newly introduced rules, by default triggering only warnings",
+            "formatting": "Related to code-style",
+            "idempotency":
+                "Possible indication that consequent runs would produce different results",
+            "idiom": "Anti-pattern detected, likely to cause undesired behavior",
+            "metadata": "Invalid metadata, likely related to galaxy, collections or roles",
+            "module": "Incorrect module usage",
+            "readability": "Reduce code readability",
+            "repeatability": "Action that may produce different result between runs",
+            "resources": "Unoptimal feature use",
+            "safety": "Increase security risk",
```

Comment:
Any reason not to call this 'unsecure' or 'security' to make it obvious what the problem could be?